cloud/google/cmd/gce-machine-controller/Makefile

```diff
@@ -18,7 +18,7 @@ GCR_BUCKET = k8s-cluster-api
 PREFIX = gcr.io/$(GCR_BUCKET)
 DEV_PREFIX ?= gcr.io/$(shell gcloud config get-value project)
 NAME = gce-machine-controller
-TAG = 0.0.9
+TAG = 0.0.10
```

Comment:
This image hasn't been built yet, I'll build/upload it once this is ready to merge.